libraries/SITL/SIM_Glider.cpp

```diff
@@ -127,7 +127,7 @@ Vector3f Glider::getTorque(float inputAileron, float inputElevator, float inputR
     float Mz = Cn * qPa * m.Sref * m.refSpan;
 
 
-#if 0
```

Comment:
This is going to be *extremely* high rate.

Are you sure you want it in?

Have you checked the SoH logs to make sure we're not just dropping vast numbers of messages with this?

Reply:
Actually, I haven't put that much thought into this line.

I saw that the logging was always disabled and thought I'd enable it, since it looks equivalent to the `AP_Logger().Write` of the `Glider::getForce` just below.

Am I missing something? Are the enclosing `if`s of this one so different?
Should we perhaps delete this inaccessible log write instead?

Reply:
I agree we need to leave it as #if 0, 1200Hz logging of AOA is not really needed